js/components/diagram/index.js

```diff
@@ -0,0 +1,39 @@
+import React, {Component} from 'react';
+import ReactHighcharts from 'react-highcharts';
+
+
+export default class Diagram extends Component {
+  constructor  (...args) {
+    super(...args);
+
+    this.state = {
+      configcharts : {
```

Comment:
you must not save this in state. Only property what will change you should store in state